index.html

```diff
@@ -2512,22 +2507,20 @@ <h3>WoT Scripting API</h3>
                 time-critical logic from the cloud down to a gateway or
                 edge node.
             </p>
-            <p> The non-normative <a>WoT Scripting API</a> document defines
-                the structure and algorithms of the programming
-                interface that allows scripts to discover, process
-                (retrieve) and produce (define) <a>Thing Descriptions</a>. The
-                implementation of the <a>WoT Scripting API</a> instantiates local
+            <p> The non-normative <a>WoT Scripting API</a> document [[?wot-scripting-api]
+                defines the structure and algorithms of the programming
+                interface that allows scripts to discover, fetch and process,
+                and produce (i.e., define and publish) <a>WoT Thing Descriptions</a>. The
```

Comment:
there is no `publish()` but `expose()`

Reply:
Maybe you can leave `publish()` and maybe we need to change `expose()` to `publish()` in the Scripting API.

Reply:
I will change to "expose". This term fits in the API as it is about exposing the Thing and not only about making its TD available (~publish).